packages/miniflare/src/workers/kv/assets.worker.ts

```diff
@@ -1,10 +1,14 @@
 import { SharedBindings } from "miniflare:shared";
-import { KVParams } from "./constants";
 
 interface Env {
 	[SharedBindings.MAYBE_SERVICE_BLOBS]: Fetcher;
+	__STATIC_ASSETS_REVERSE_MAP: AssetReverseMap;
```

Comment:
By the way there is no need to use special naming here (e.g. `__` prefix) since this binding is already private to this fake KV namespace service.